drivers/display/Kconfig.sdl

```diff
@@ -0,0 +1,45 @@
+# Kconfig - SDL based emulated display configuration options
+
+#
+# Copyright (c) 2018 dXplore
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig SDL_DISPLAY
+	bool "SDL based emulated display"
+	depends on SOC_POSIX
+	default n
+	select HAS_SDL
```

Comment:
The farther I go thru the code the mode I feel the "`HAS_SDL` HW model" is really a piece of the driver.